features/users/insert-all-users.ts

```diff
@@ -0,0 +1,25 @@
+const { userColumns } = require('../../db/user-columns');
+
```

Comment:
If I would compose this function with others, it would not work:
`insertAllUsersAsInLisbon(..,..).then(...`

Also, why would we code the name of the city in the function, instead of passing the name of the city as an input parameter to the function?